great_expectations/data_context/data_context/base_data_context.py

```diff
@@ -1077,7 +1075,7 @@ def save_profiler(
         key: Union[GeCloudIdentifier, ConfigurationIdentifier]
         if self.ge_cloud_mode:
             key = GeCloudIdentifier(
```

Comment:
Can we change the name of GeCloudIdentifier, too?

Reply:
hmm as long as Cloud doesn't directly use this! I don't think they though but lemme peruse the Mercury code first